deployment/grafana/02-grafana-configmap.yaml

```diff
@@ -457,7 +1178,7 @@ data:
           "thresholds": [],
           "timeFrom": null,
           "timeShift": null,
-          "title": "Process Queue Size",
+          "title": "Kubernetes Process Queue Size",
```

Comment:
I wonder if there's a better title we can use for this. Maybe `Gimbal Replication Queue Size`?

This metric is tracking how many services/endpoints are in the queue for being replicated/synced/discovered into the gimbal cluster.